azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherActivity.kt

```diff
@@ -26,6 +29,14 @@ class CallLauncherActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        if (!AppCenter.isConfigured() && !BuildConfig.DEBUG) {
+            AppCenter.start(
+                application,
+                "8cb48e80-76ae-43f4-939a-ab6edd255af4",
```

Comment:
We should not hardcode app id here.
Let's make it as a part of the build process. When assembling and app in the pipeline we can populate local properties. Also this way we will avoid posting crash logs happened while developing on the local machine.